### DISCUSSION TITLE
About Traditional Chinese model

### DISCUSSION THREAD

**Discussion by pactera-song** [explosion/spaCy#6025] [Help: Other Questions] [answered]
Does spaCY support traditional Chinese? ZH-TW or ZH-HK.

**Comment by adrianeboyd** [comment, marked as answer]
In the basic `Chinese` language support, there's not much specific to simplified Chinese, except maybe the stop words. If you have a `jieba` dictionary or `pkuseg` model for traditional Chinese characters, it should work fine.

The provided Chinese models like `zh_core_web_sm` are trained on OntoNotes 5, which only contains simplified Chinese (see p. 27 in the docs: https://catalog.ldc.upenn.edu/docs/LDC2013T19/OntoNotes-Release-5.0.pdf). If you know of data with a permissive license that can be used to train models for traditional Chinese (typically it's hardest to find NER data), we'd be happy to look into whether we could provide additional models.
